DuckDuckGo/DoNotSellSettingsViewController.swift

```diff
@@ -0,0 +1,110 @@
+//
+//  DoNotSellSettingsViewController.swift
+//  DuckDuckGo
+//
+//  Created by Brad Slayter on 9/18/20.
+//  Copyright © 2020 DuckDuckGo. All rights reserved.
+//
+
+import UIKit
+import Core
+
+class DoNotSellSettingsViewController: UITableViewController {
+
+    @IBOutlet var labels: [UILabel]!
+    
+    @IBOutlet weak var doNotSellToggle: UISwitch!
+    @IBOutlet weak var headerView: UIView!
+    @IBOutlet weak var footerView: UIView!
```

Comment:
Table headers and footers have been ... problematic for the translations.  Once you fix the hardcoded string problem, can you please test with a pseudo language enabled (instructions in another comment) and check different screen sizes and especially rotation.  Again, please ping @bwaresiak for help here.

Reply:
Tested this with a few different pseudo languages and appears to be displaying properly.